<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ch_pass</class>
 <widget class="QDialog" name="ch_pass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	
	background-image: url(:/img/img/hero-background-15.jpg);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Mot de passe actuel:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="curr_pass">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Nouveau mot de passe:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="new_pass">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Confirmer nouveau mot de passe:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="r_new_pass">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #fff;
	color:  #000;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>300</y>
     <width>306</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="comfirm">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	height: 25px;
	font: 81 11pt &quot;Fira Sans Compressed ExtraBold&quot;;
}
</string>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	height: 25px;
	font: 81 Italic 11pt &quot;Fira Sans Compressed ExtraBold&quot;;
	color: black;
	background-color: red;
}

QpushButton:pressed{
	background-color: rgb(182, 7, 7);
}</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
